protocols/server/cloud-events/main.tsp

```diff
@@ -0,0 +1,271 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+using TypeSpec.Http;
+
+/**
+ * This event is triggered when the service receive an MQTT CONNECT packet.
+ */
+@route("OnMqttClientConnect/Doesn'tMatter/DependOnYourEventHandlerSettings")
+op OnMqttConnect(
```

Comment:
if the upstream server expects both mqtt and default upstream events, what would be the field for the server to determine the type of the incoming request? Do we need some field like `Connection-Type: mqtt/default` in the request header?

Reply:
Or do we also provide an instruction how to do routing, e.g. `@route(/eventhandler/mqtt/events/connect)`

Reply:
Will add a new header to indicate client type.

Reply:
can typespec also describe the expected header?

Reply:
Yes, there is already a class called "CommomRequestheader"